src/audio/volume/volume.c

```diff
@@ -184,6 +192,7 @@ const struct comp_zc_func_map zc_func_map[] = {
 #endif /* CONFIG_FORMAT_S32LE */
 };
 
+#if CONFIG_IPC_MAJOR_3
```

Comment:
It would be nicer if there would be kconfig for linear ramp enable and there is now for windows fade. If IPC4 is not capable to specify ramp type disable of other but windows fade could be done via kconfig.